front/app/containers/Admin/pages-menu/containers/HeroBannerForm/HeaderImageDropzone.tsx

```diff
@@ -22,7 +19,8 @@ const HeaderImageOverlay = styled.div<{
 `;
 
 interface Props {
-  latestAppConfigStyleSettings?: IAppConfigurationStyle;
+  homepageSettingsOverlayOpacity: number;
+  homepageSettingsOverlayColor: string;
```

Comment:
To make it shorter, it could just be overlayOpacity and overlayColor as prop names here, within the HeaderImageDropzone context, it's not important to know it's about the homepageSettings from the prop names.

Reply:
👍 super helpful, I was actually wondering if there was a way to do something like this in TS

Reply:
For the second comment, I agree but later in the file we use those variable names to get the final value from either settings or theme:

```
  const overlayColor = homepageSettingsOverlayColor ?? theme.colorMain;
  const overlayOpacity =
    homepageSettingsOverlayOpacity ?? theme.signedOutHeaderOverlayOpacity;
```